package.json

```diff
@@ -73,7 +73,8 @@
     "jest": "^24.8.0",
     "jest-environment-jsdom-fourteen": "^0.1.0",
     "lodash": "^4.17.10",
-    "pixi.js": "4.8.8",
+    "pixi.js": "^5.0.4",
+    "pixi.js-legacy": "^5.0.4",
```

Comment:
if you want more tests to passe, change State.js import by: `import { Application } from 'pixi.js-legacy'` but try to run rollup: it will fail.

Reply:
I fix rollup build to use pixi.js-legacy